src/pages/Missions.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { getMissions } from '../redux/missions/missionsslice';
+import { getMissions, changeMissionStatus } from '../redux/missions/missionsslice';
 
 const Missions = () => {
   const dispatch = useDispatch();
   const missions = useSelector((state) => state.missionsreducer.missions);
   if (missions.length === 0) {
     dispatch(getMissions());
   }
```

Comment:
This is the approach I refer to, this only fires the first time the missions page is rendered.